### ISSUE TITLE
Insufficient coins for testnet

### ISSUE THREAD

**Issue by ooffeerr** [bitcoin-wallet/bitcoin-wallet#333]
Whenever trying to perfrom a TestNet transaction I'm getting this error message:
"Not enough available coins. You're missing <Some BTC amount>"

Is working working with TestNet expected to work somehow?

**Comment by schildbach** [comment]
How many coins do you have on your wallet? Be aware that you always need to pay the network fee.

**Comment by ooffeerr** [comment]
Thanks for the reply.
I'm probably missing something very trivial, but I don't think I understand how I set/add my own address.
I've used two devices - one set to request Bitcoin, with the QR code, and another to send Bitcoin, and I've scanned the QR code before.

I've used this tutorial: https://bitcoin.org/en/developer-examples#testing-applications
And this: https://tpfaucet.appspot.com/    to send test bitcoin to the addresses I've tried to send and receive to.

After scanning the QR code at the sender device, whenever I'm changing the amount to send, the TextView with the above error is refreshed with the same error message, regardless of what I'm trying to send...
Any idea?

Thanks again.
